src/chunkserver/spdk_hook.h

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright (c) 2020 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: curve
+ * Created Date: 20221103
+ * Author: xuchaojie
+ */
+
+#ifndef SRC_CHUNKSERVER_SPDK_HOOK_H_
+#define SRC_CHUNKSERVER_SPDK_HOOK_H_
+
+#include <spdk/log.h>
+#include <rte_log.h>
+#include <rte_malloc.h>
+#include <pfs_spdk.h>
+#include <pfs_api.h>
+#include <pfs_trace_func.h>
+#include <pfsd.h>
+#include <pfsd_sdk.h>
+#include <pfsd_sdk_log.h>
+
+void glog_pfs_func(int level, const char *file, const char *func, int line,
```

Comment:
did you forget to put these functions in a namespace?

Reply:
no, I think we should put this in global namespace